CMakeLists.txt

```diff
@@ -41,6 +41,13 @@ message(STATUS "USE_PYTEST_HELPER_BINDING: ${USE_PYTEST_HELPER_BINDING}")
 if(BUILD_QT)
     option(QT3D_USE_RHI "Qt use RHI" OFF)
     message(STATUS "QT3D_USE_RHI: ${QT3D_USE_RHI}")
+endif()
+
+# PUI-related code has been migrated to PySide6, so it doesn't need to be installed by default.
+# Yet, you can still install it by setting the INSTALL_PUI option to ON.
+option(INSTALL_PUI "install PUI" OFF)
+message(STATUS "INSTALL_PUI : ${INSTALL_PUI}")
+if(INSTALL_PUI)
```

Comment:
Please check if `BUILD_QT` is turned on, and issue a warning message if it is not, because installing PUI without Qt GUI built is useless.

Reply:
Do not resolve comments.  Resolving comments makes it hard to track.